src/main/java/emissary/client/HTTPConnectionFactory.java

```diff
@@ -96,7 +95,8 @@ private HTTPConnectionFactory() {
             final Configurator cfg = config == null ? ConfigUtil.getConfigInfo(HTTPConnectionFactory.class) : config;
             // if someone doesn't want keep alives...
             if (!cfg.findBooleanEntry(CFG_HTTP_KEEPALIVE, DFLT_KEEPALIVE)) {
-                this.connReuseStrategy = NoConnectionReuseStrategy.INSTANCE;
+                this.connReuseStrategy = DefaultConnectionReuseStrategy.INSTANCE;
```

Comment:
This change is probably fine, but I'm just noting here.
We're moving from a "A strategy that never re-uses a connection" posture to a more subtle re-use policy, where sometimes connections will be reused.  Again, I think it's probably safe to make this change but if it introduces problems, they will probably be tricky to track down to this change.  The strategy we changed to does the following:

```
Default implementation of a strategy deciding about connection re-use. The strategy determines whether a connection is persistent or not based on the message’s protocol version and Connection header field if present. Connections will not be re-used and will close if any of these conditions is met

- the close connection option is present in the request message
- the response message content body is incorrectly or ambiguously delineated
- the close connection option is present in the response message
- If the received protocol is HTTP/1.0 (or earlier) and keep-alive connection option is not present

In the absence of a Connection header field, the non-standard but commonly used Proxy-Connection 
header field will be used instead. If no connection options are explicitly given the default policy for the 
HTTP version is applied.  HTTP/1.1 (or later) connections are re-used by default. HTTP/1.0 (or earlier)
 connections are not re-used by default.
```